cmd/root.go

```diff
@@ -22,39 +22,48 @@ THE SOFTWARE.
 package cmd
 
 import (
-	"fmt"
 	"os"
 	"starq/internal/starq"
 
 	"github.com/spf13/cobra"
 )
 
-// rootCmd represents the base command when called without any subcommands
-var rootCmd = &cobra.Command{
-	Use:   "starq [-r <jq-rule> ...] [-c <config-file>.yaml] [-a <jq-rule> ...]",
-	Short: "starq transforms a document from stdin using jq-style rules.",
-	Long:  `starq (pronounced "star-q") transforms a JSON or YAML document from stdin using jq-style rules and outputs to stdout.`,
-	Run: func(cmd *cobra.Command, args []string) {
-		err := cmd.ParseFlags(args)
-		exitIf(err)
+// NewRootCmd creates a new instance of the base command without any subcommands
+func NewRootCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "starq [<config-file>.yaml...] [-r <jq-rule> ...] [-a <jq-rule> ...]",
```

Comment:
I am assuming r is prepend and a is append, but I am not understanding what is being prepended and appended and to what. I think you need more description.